lib/mocha.js

```diff
@@ -402,6 +402,18 @@ Mocha.prototype.noHighlighting = function() {
 };
 
 /**
+ * Enable uncaught errors to propagate (in browser).
+ *
+ * @return {Mocha}
+ * @api public
+ */
+
+Mocha.prototype.allowUncaught = function(){
+  this.options.allowUncaught = global.mocha.allowUncaught = true;
```

Comment:
Hmm, taking a closer look, I don't like this.  We should be handling this option the same way we handle others.

```js
Mocha.prototype.allowUncaught = function(){
  this.options.allowUncaught = true;
  return this;
};
```

Below, when you need to read the setting, you'll need to get it a different way.